cfgov/jinja2/v1/_includes/molecules/line-chart.html

```diff
@@ -0,0 +1,41 @@
+
+{# ==========================================================================
+
+   Chart
+
+   ==========================================================================
+
+   Description:
+
+   Create a line chart molecule with data.
+   See [GHE]/flapjack/Modules-V1/wiki/Chart
+
+   ========================================================================== #}
+
+<div class="chart_wrapper block">
+    {% if value.title %}
+        <h3>{{ value.title }}</h3>
+    {% endif %}
+
+	<div class="svg_wrapper"
+        id="{{ value.element_id }}"
+        data-chart-type="{{ value.chart_type }}"
+        data-source="{{ value.data_source }}"
+        data-title="{{ value.title }}"
+        data-market="{{ value.market }}"
+        data-report-type="{{ value.report_type }}"
+    ></div>
+
+	<p class="block__sub block__border-top block short-desc chart_footnote">
+		{% if value.data_source %}
+        <strong>Source:</strong> <a class="icon-link icon-link__download" href="{{ parse_links(value.data_source) | safe }}"> <span class="icon-link_text">Download CSV file</span> </a><br>
```

Comment:
Why does this need a `parse_links` and `| safe` call, if this is just a URL that's being passed in?